### PR TITLE
Displays Individual Reminder as an output to a reminder link 

### PR DESCRIPTION
**Pull request by Pilewski**
## Purpose

-When the user clicks on one of the reminders on the page, they are taken to an individual reminders route. The title is the link. And the output of the link is the full reminder.
-Also had to change our faker method because Ian's random words were not cooperating 
-Closes Issue #3 

## Approach

-Nested a route of  `this.route('reminder', { path: ':reminder_id' })` inside of the reminders route.
-Created two directories of reminders inside of our routes and templates folders. 
-Reminder route finds reminder by id
-Reminder template renders the title, date and notes

### Learning

-Ember docs

#### After

@martensonbj @brittanystoroz @TMan22

### CODE REVIEW

File: app/routes/reminders.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   model() {
-    return this.get('store').findAll('reminder')
+    return  this.get('store').findAll('reminder');
```
Comment (nfosterky) [inline app/routes/reminders.js]: like the semi-colon, get rid of the extra space before `this.get`

File: app/routes/reminders/reminder.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.find('reminder',params.reminder_id)
```
Comment (nfosterky) [inline app/routes/reminders/reminder.js]: add an extra space after the `,` to make it easier to read and for consistency

File: app/templates/reminders.hbs [hunk 3]
```diff
@@ -1,9 +1,8 @@
 <ul class="spec-reminder-item">
   {{#each model as |reminder|}}
   <article>
-    <h2>{{reminder.title}}</h2>
+    <h2>{{#link-to 'reminders.reminder' reminder}}{{reminder.title}}{{/link-to}}</h2>
```
Comment (nfosterky) [inline app/templates/reminders.hbs]: There is a lot happening on this line, split into multiple lines

**Comment by martensonbj** [comment]
What's the status on these changes?

File: app/templates/reminders.hbs (hunk 3 above)
Comment (ianlancaster) [inline app/templates/reminders.hbs]: I see what you mean. Done.

**Comment by ianlancaster** [comment]
All requested changes have been made @martensonbj @nfosterky 👍

**Pull request merged**